plugins/vendor/zoom/lib/api/webPage/KalturaZoomRegistrationPage.html

```diff
@@ -1,136 +1,49 @@
-        }
-
-        /* Avatar image */
-        img.avatar {
-            width: 20%;
-        }
-
-        /* Add padding to containers */
-        .container {
-            padding: 10px;
-        }
-
-        label {
-            font-family: Arial, Helvetica, sans-serif;
-        }
-
-    </style>
+    <title>Kaltura Zoom connector</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>
     <script>
         var ks = '@ks@';
-        var url = '@BaseServiceUrl@/api_v3/service/vendor_zoomvendor/action/submitRegistration';
-        var accountId = '@accountId@';
+        let baseServiceUrl = '@BaseServiceUrl@';
+        var accountId = '@accoutId@';
+        var partnerId= '@partnerId@';
+        var submitUrl = baseServiceUrl+'/api_v3/service/vendor_zoomvendor/action/submitRegistration';
+        var getUrl = baseServiceUrl + '/api_v3/service/vendor_zoomvendor/action/get?format=1';
+
     </script>
     <script>
-        function trySubmit(event) {
+        function trySubmit() {
             $("body").css("cursor", "progress");
-            event.preventDefault();
-            var $defaultUserId = $('#defaultUserId');
-            var $zoomCategory  = $('#zoomCategory');
-            var $zoomWebinarCategory  = $('#zoomWebinarCategory');
-            var defaultUserId = $defaultUserId.val().trim();
-            var zoomCategory = $zoomCategory.val().trim();
-            var zoomWebinarCategory = $zoomWebinarCategory.val().trim();
-            var enableRecordingUploadObj = $(".enableRecordingUpload");
-            var enableWebinarUploadObj = $(".enableWebinarUpload");
-            var createUserIfNotExistObj = $(".createUserIfNotExist");
-            var enableRecordingUpload = enableRecordingUploadObj.is(":checked");
-            var enableWebinarUpload = enableWebinarUploadObj.is(":checked");
-            var createUserIfNotExist = createUserIfNotExistObj.is(":checked");
-            var handleParticipantsMode = $('[name=participantsHandler]:checked').val();
-            var zoomUserMatchingMode = $('[name=userMatchingHandler]:checked').val();
-            var zoomUserPostfix = $('#zoomUserPostfix').val().trim();
+            let defaultUserId = $('#defaultUserId').val().trim();
+            let zoomCategory  = $('#zoomCategory').val().trim();
+            let zoomWebinarCategory  = $('#zoomWebinarCategory').val().trim();
+            let enableRecordingUpload = $("#enableRecordingUpload").is(":checked");
+            let enableWebinarUpload = $("#enableWebinarUploads").is(":checked");
```

Comment:
it looks like it was like that before, but var and later the title called enableWebinarUpload but the object called with uploads